scripts/cmake/cmake/FindMKL.cmake

```diff
@@ -0,0 +1,69 @@
+# Find Intel Math Karnel Library (MKL)
```

Comment:
Either  a source of this file, or our copyright message is missing. (Though the other cmakes don't have it too, but let's start here.)